dev/io.openliberty.pages.3.1.internal_fat/fat/src/io/openliberty/pages31/fat/FATSuite.java

```diff
@@ -33,8 +33,8 @@
  */
 @RunWith(Suite.class)
 @SuiteClasses({
-    TagsTestsEE10.class,
-    UnknownELIdentifiersTest.class,
+    // TagsTestsEE10.class,
+    // UnknownELIdentifiersTest.class,
```

Comment:
I assume this was just for debug locally and should be uncommented?

Reply:
Oops fixed